From 63b5edeb289b8b0c988c3e4f3657198a8dd7af24 Mon Sep 17 00:00:00 2001
From: Sean Kau <skau@chromium.org>
Date: Thu, 8 Jul 2021 16:00:28 -0700
Subject: [PATCH] Set the counted pages for PDFs as job-impressions

pwgraster is always == 0 so the code which sets job-impressions
never runs. Remove the test condition to ensure this code runs
for both pwg and PDFs.

[Adrian Ratiu: rebased to v1.28.17]
---
 filter/gstoraster.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/filter/gstoraster.c b/filter/gstoraster.c
index 87ff948e6..42dd3f42c 100644
--- a/filter/gstoraster.c
+++ b/filter/gstoraster.c
@@ -761,10 +761,8 @@ main (int argc, char **argv, char *envp[])
     goto out;
   }
 
-  if (pwgraster) {
     // Set job-impressions for later embedding as TotalPageCount.
     num_options = cupsAddIntegerOption("job-impressions", pagecount, num_options, &options);
-  }
 
   if (argc == 6) {
     /* input from stdin */
-- 
2.39.2

